File: backend/src/main/java/org/jdk/project/dto/statistics/PassengerStatisticsQuery.java
package org.jdk.project.dto.statistics;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class PassengerStatisticsQuery {
  private String roomCode;
  private String flightDate;
  private String cardNo;
  private String accessType;
  private String status;
}

File: backend/src/main/kotlin/org/jdk/project/dto/statistics/PassengerStatisticsQuery.kt
package org.jdk.project.dto.statistics

data class PassengerStatisticsQuery(
    var roomCode: String? = null,
    var flightDate: String? = null,
    var cardNo: String? = null,
    var accessType: String? = null,
    var status: String? = null,
)
